### ISSUE TITLE
Use custom Vue components for rendering cells

### ISSUE THREAD

**Issue by dgaus** [grid-js/gridjs-vue#44]
Is there any support planned for this? It would be nice to be able to specify a component with a defined interface as a renderer

**Comment by selfagency** [comment]
Meet the new `render` function, now available in v0.5.x:

```
import FormatterComponent from './FormatterComponent.vue'
 
[...]
 
this.cols = [
  {
    name: 'Model',
    formatter: cell => {
      const current = this.$gridjs.uuid()
      this.$gridjs.render(`[data-ref="${current}"]`, FormatterComponent, { content: cell, otherProp: true })
      return this.$gridjs.html(`<div data-ref="${current}"></div>`)
    }
  },
  { ... },
  { ... }
]
```

**Comment by alexis-karyavin** [comment]
Not working with 5 version grid.js and vue3
